curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a417f7f9daf2e9aae6831.md

```diff
@@ -36,7 +36,7 @@ In which statement does the term CEO refer to a specific person known to both th
 
 ### --feedback--
 
-`a CEO` means it could be any CEO, not someone they both know. Try again
+`a CEO` means it could be any CEO, not someone they both know. 
```

Comment:
```suggestion
`a CEO` means it could be any CEO, not someone they both know.
```
Nitpick, there's leftover trailing space at the end of line. Please remove it here and in other cases where it exist after the changes.